### DISCUSSION TITLE
11월 첫째 주 스프린트 [11/1 - 11/7]

### DISCUSSION THREAD

**Discussion by ghost** [team-mokcha/corabal-beta#39] [스프린트 일지]
## 브랜치 38 현재까지의 성과
- 메인, 로그인 화면은 리팩토링 완료 (메인 세부 컴포넌트는 조금 수정해야 함 - @alveloper)
- 폴더구조 리팩토링이 완료되고 나면 css 고치기

## 다음주 계획
- @HyejinYang, @alveloper : 닉네임, 팝업 모달(에러, 목표 설정, 구매 등) 리팩토링
  - 타입을 뭘 받는 거에 따라서 뭘 넘겨주는지 정하고 그 템플릿 안에서 타입에 따라서 받아오는 거 고쳐주기
- @alveloper : 위 리팩토링이 생각보다 빨리 끝난다면 메인 세부 컴포넌트들 고치기 (type 지정 등등)

**Comment by HyejinYang** [comment]
* 차주 작업은 수(11월 10일)부터 시작
